components/configurations-generator/Jenkinsfile

```diff
@@ -54,19 +54,19 @@ podTemplate(label: label,
                         }
 
                         stage("install dependencies $application") {
-                            execute("make resolve", workDir, componentDir)
+                            execute("dep ensure --vendor-only -v", workDir, componentDir)
                         }
 
                         stage("code quality $application") {
-                            execute("make validate", workDir, componentDir)
+                            execute("gometalinter --vendor --deadline=2m --disable-all --enable=vet --skip=\$(go list ./... | grep \"/fake\") ./...", workDir, componentDir)
                         }
 
                         stage("build $application") {
                             execute("CGO_ENABLED=0 go build -o bin/app cmd/generator/main.go", workDir, componentDir)
                         }
 
                         stage("test $application") {
-                            execute("make test-report", workDir, componentDir)
+                            execute("2>&1 go test -short -v ./... | go2xunit -fail -output unit-tests.xml", workDir, componentDir)
```

Comment:
Why -short flag is present here? If it means we are skipping some tests we need to understand why and find a way to reenable them.

Reply:
Here's an example of such test: https://github.com/kyma-project/kyma/blob/master/components/api-controller/pkg/controller/networking/v1/networkingv1_integration_test.go#L17. If yo ucopied this from api-controller there may be no such test in configurations-generator, but please verify anyway.